build/package/Installer.DEB.proj

```diff
@@ -95,11 +95,11 @@
 
       <!-- Remove Shared Framework and Debian Packages -->
       <Exec Command="sudo dpkg -r $(SdkDebianPackageName)" />
-      <Exec Command="sudo dpkg -r $(AspNetCoreSharedFxDebianPackageName)" />
-      <Exec Command="sudo dpkg -r $(SharedFxDebianPackageName)" />
+      <Exec Command="sudo dpkg -r $(AspNetCoreSharedFxDebianPackageFileName)" />
```

Comment:
We should double check that this works with the new package names. The package installed is aspnetcore-runtime-2.1 but I think this property is set to aspnetcore-runtime-2.1.0-rc1-30661 which is the non-upgradeable installer package name.